mzroll/mainwindow.cpp

```diff
@@ -1639,6 +1656,119 @@ void MainWindow::loadCompoundsFile() {
 	}
 }
 
+// open function for set csv
+void MainWindow::loadSetsFile() {
+	LOGD;
+	QStringList filelist =
+			QFileDialog::getOpenFileNames(this, "Select Set Information File To Load",
+					".",
+					"All Known Formats(*.csv *.tab *.tab.txt);;Tab Delimited(*.tab);;Tab Delimited Text(*.tab.txt);;CSV File(*.csv)");
+
+    if ( filelist.size() == 0 || filelist[0].isEmpty() ) return;
+	if(!loadSetsFile(filelist[0])) {
+		string dbfilename = filelist[0].toStdString();
+		string dbname = mzUtils::cleanFilename(dbfilename);
+		string notFoundColumns = "Following are the unknown column name(s) found: ";
+
+		QMessageBoxResize msgBox;
+		msgBox.setText(tr("Trouble in loading set information %1").arg(QString::fromStdString(dbname)));
+		msgBox.setIcon(QMessageBoxResize::Warning);
+		if (DB.notFoundColumns.size() > 0) {
```

Comment:
DB is for the compound database. You can't use this condition here for loading sets.